common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -5923,40 +5923,40 @@
       "Failed to truncate table <tableIdentWithDB> when removing data of the path: <path>."
     ]
   },
-  "_LEGACY_ERROR_TEMP_1270" : {
-    "message" : [
-      "SHOW CREATE TABLE is not supported on a temporary view: <table>."
-    ]
-  },
-  "_LEGACY_ERROR_TEMP_1271" : {
-    "message" : [
-      "Failed to execute SHOW CREATE TABLE against table <table>, which is created by Hive and uses the following unsupported feature(s)",
-      "<unsupportedFeatures>",
-      "Please use `SHOW CREATE TABLE <table> AS SERDE` to show Hive DDL instead."
```

Comment:
Whether `AS SERDE` is specified or not, an exception will be thrown, such as in the test case `hive partitioned view is not supported` of `HiveSQLViewSuite`, so this line is unnecessary, and merged `_LEGACY_ERROR_TEMP_1271 ` and `_LEGACY_ERROR_TEMP_1275 `.